src/cascadia/WindowsTerminal/main.cpp

```diff
@@ -73,6 +73,11 @@ static void EnsureNativeArchitecture()
     }
 }
 
+static bool _messageIsF7Keypress(const MSG& message)

```

Comment:
I don't really care that there's not a doc comment here, seems self-explanatory

Reply:
THANK YOU